### PR TITLE
Replace cluster with addCluster 

### PR DESCRIPTION
**Pull request by theotimeee**
After mia PR https://github.com/microbiome/mia/pull/502 renaming `cluster` to `addCluster` in the mia package, this PR replaces `cluster` with `addCluster` in the miaViz package.

### CODE REVIEW

File: R/plotDMN.R [hunk 1]
```diff
@@ -54,8 +54,8 @@ setMethod("plotDMNFit", signature = c(x = "SummarizedExperiment"),
         if (!is.null(metadata(x)[[name]]$dmm)) {
             dmn <- metadata(x)[[name]]$dmm
         } else {
-            .Deprecated(old="getDMN", new="cluster", 
-                    "Now runDMN and calculateDMN are deprecated. Use cluster with DMMParam parameter and full parameter set as true instead.")
+            .Deprecated(old="getDMN", new="addCluster", 
+                    "Now runDMN and calculateDMN are deprecated. Use addCluster with DMMParam parameter and full parameter set as true instead.")
```
Comment (antagomir) [inline R/plotDMN.R]: could be splitted on two rows.

**Review by antagomir — APPROVED** [review]
Seems good to me overall.

**Comment by antagomir** [comment]
Conflicts to resolve.

**Review by TuomasBorman — APPROVED** [review]

**Comment by antagomir** [comment]
up

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**